be/src/storage/rowset/index_page.cpp

```diff
@@ -113,13 +114,15 @@ Status IndexPageIterator::seek_at_or_before(const Slice& search_key) {
     return Status::OK();
 }
 
+// This function has the meaning of interval, in fact, it is to find first possible Page which >=search key
 Status IndexPageIterator::seek_at_or_after(const Slice& search_key) {
```

Comment:
Why the `seek_at_or_before` and `seek_at_or_after` methods  aren't symmetrical?

Reply:
Fixed